woocommerce/woocommerce-android
# [Order Creation] Introduce Any Variation support to Bundles

Summary
==========
Fix issue #11761 by introducing a strategy where when we detect an Any selection, the app just sets a given possible option as the attributes selection instead of Any so the API accepts the Order creation, and once the Order is created, everything is precisely as the user selected. It’s a weird approach regarding code logic, but that’s because the Bundles API doesn’t accept the Any selection. For the user, though, it looks way more natural.

Screen Capture
==========
https://github.com/user-attachments/assets/03443a32-075e-4d07-bcf8-20241a0d4649

How to Test
==========
1. Start the order creation
2. Add a bundled product which contains a variable product that has all the variations set with Any for the attributes
3. Tap to Select Variation
4. Notice all the Variation rows loads correctly.
5. Select one of the rows where all attributes are set as `Any`
6. Add the product to the order
7. Notice the Bundled Product loads correctly, but the children loads as expected.

Update release notes:


## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
